test/azureServiceClientTests.ts

```diff
@@ -2175,9 +2175,9 @@ describe("AzureServiceClient", () => {
           const httpRequest = new WebResource("https://fake.azure.com/longRunningOperation", "POST");
```

Comment:
Please remove the last response from this test's service client (lines 2168 through 2173).